tests/unittest/auto_deploy/singlegpu/smoke/test_ad_speculative_decoding.py

```
@@ -155,6 +202,36 @@ def test_mtp_autodeploy_uses_eagle_one_model_capture():
 
 
+def test_eagle_quant_config_remaps_excludes_from_drafter_mapping():
+    class TargetFactory:
+        def get_quant_config(self):
+            return {
+                "quant_algo": "NVFP4",
+                "exclude_modules": ["mtp.*", "mtp*", "lm_head"],
+            }
+
+    class DraftFactory:
+        _quant_exclude_conversion_mapping = {r"^mtp(?=\.|\*)": "model"}
```

Ruff (0.15.14)

[warning] Mutable default value for class attribute

(RUF012)